scripts/npm_scripts/generateCerts.js

```diff
@@ -26,7 +24,8 @@ module.exports.installAndGenerateCertsamd64 = async function() {
 };
 
 module.exports.installAndGenerateCertsMac = async function() {
-	await runShellCommand(`curl --create-dirs --output ${binariesPath}/${darwinTarFile} ${darwinBinaries}`, null);
+	await runShellCommand(`mkdir -p ${binariesPath}`, null);
+	await runShellCommand(`wget ${darwinBinaries} -P ${binariesPath}`, null);
```

Comment:
`wget` does not exist on Macs by default, which is why the existing version uses `curl` instead

Reply:
I forgot I had changed this here. Switching to wget was meant to solve a problem we had been seeing with downloading the Fabric binaries when developers were in areas remote to GitHubs CDN. We had made the switch in Fabric for the bootstrap script and Gari had had us add wget as a prereq for Fabric. We quickly found out though, depending on which version of wget you have, they don't handle URLs generated from variables well. We need to revert our change in Fabric, and here as well. I'll switch it back to curl. Thanks for highlighting this.

Reply:
Fixed, had to add the `-L` flag to handle redirects, tested to work